core/src/main/scala/org/apache/spark/deploy/mesos/ui/MesosClusterPage.scala

```diff
@@ -76,6 +104,7 @@ private[mesos] class MesosClusterPage(parent: MesosClusterUI) extends WebUIPage(
       <td>{state.startDate}</td>
       <td>{state.slaveId.getValue}</td>
       <td>{stateString(state.mesosTaskStatus)}</td>
+      {sandboxCol}
```

Comment:
Not a big deal, but it looks like all the other cells have `<td>` elements in here, and the call just generates the contents of that cell. I'd prefer to keep the same pattern everywhere.